system_files/desktop/shared/usr/share/ublue-os/just/84-bazzite-virt.just

```diff
@@ -41,29 +41,30 @@ setup-virtualization ACTION="":
       )
     fi
     if [[ "${OPTION,,}" =~ (^enable[[:space:]]virtualization|virt-on) ]]; then
-      virt_test=$(rpm-ostree status | grep -A 4 "●" | grep "virt-manager")
-      if [[ -z ${virt_test} ]]; then
-        echo "Installing QEMU and virt-manager..."
-        rpm-ostree install -y virt-manager edk2-ovmf qemu
-        rpm-ostree kargs \
-        --append-if-missing="kvm.ignore_msrs=1" \
-        --append-if-missing="kvm.report_ignored_msrs=0"
-        echo 'Please re-run "ujust setup-virtualization" after the reboot to enable libvirtd service'
-      fi
-    elif [[ "${OPTION,,}" =~ (^disable[[:space:]]virtualization|virt-off) ]]; then
-      virt_test=$(rpm-ostree status | grep -A 4 "●" | grep "virt-manager")
-      if [[ ${virt_test} ]]; then
-        if [ "$(systemctl is-enabled libvirtd.service)" == "enabled" ]; then
-          echo "${red}Disabling${n} libvirtd before removal"
-          sudo systemctl disable --now libvirtd 2> /dev/null
+      (
+        virt_test=$(rpm-ostree status -v --jsonpath '.deployments[0].packages')
+        if [[ ${virt_test} == *virt-manager* ]]; then
+          echo "Installing QEMU and virt-manager..."
+          rpm-ostree install -y virt-manager edk2-ovmf qemu
+          rpm-ostree kargs \
+          --append-if-missing="kvm.ignore_msrs=1" \
+          --append-if-missing="kvm.report_ignored_msrs=0"
+          sudo systemctl enable bazzite-libvirtd-setup.service \
```

Comment:
Think you might have forgotten to disable this when you disable virtualization, other than that it looks good.